docsrc/content/parsing.fsx

```diff
@@ -0,0 +1,77 @@
+(*** hide ***)
+// This block of code is omitted in the generated HTML documentation. Use 
+// it to define helpers that you do not want to show in the documentation.
+#I "../../bin"
+open System
+#r @"../../src/FSharpPlus/bin/Release/net45/FSharpPlus.dll"
+open FSharpPlus
+(**
+# Parsing
+
+F#+ provides several helper methods in order to simplify building parsers and parsing like tasks. 
+*)
+
+
+(**
+## sscanf, trySscanf and friends
+
+In f# you have some nice utility functions for creating printf style string writer function. In f#+ we find the inverse: sscanf and trySscanf.
```

Comment:
Let's be consistent with the rest of the docs and use F# (uppercase) instead of f#
Same goes for F#+ and C#